test/index.js

```diff
@@ -3,6 +3,14 @@ import FullStory from '../src';
 
 const testOrg = '123';
 
+beforeEach(() => {
+  if (window[window._fs_namespace]) {
+    delete window._fs_initialized;
+    delete window[window._fs_namespace];
+    delete window._fs_namespace;
+  }
+});
+
```

Comment:
@craigrmccown related to my comment above - this is how I'm resetting the initialization state between tests.